src/Component/RuleGenerator/RuleGenerator.example.md

```diff
@@ -1,8 +1,7 @@
 This demonstrates the use of `RuleGenerator`.
 
 ```jsx
-const React = require('react');
-const { RuleGenerator } = require('../../index');
```

Comment:
Even if not needed for styleguide itself, I would suggest to import all related classes for better real world appliance of the examples.

Reply:
I had to remove it, as i got 'already defined' errors when leaving the import in the example.